github: update PR template (#10866)
VictoriaMetrics/VictoriaMetrics: .github/pull_request_template.md

**PLEASE REMOVE LINE BELOW BEFORE SUBMITTING**

> Before creating the PR, verify that you read and followed [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/victoriametrics/contributing/#pull-request-checklist).

Reviewer: ```suggestion
Before creating the PR, make sure you have read and followed the [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/victoriametrics/contributing/#pull-request-checklist).
```
